### DISCUSSION TITLE
The cycle was running incorrectly according to the cron expression

### DISCUSSION THREAD

**Discussion by guytechNet** [quartznet/quartznet#2279] [Q&A]
I would like a task to start running at minute 7 and then run every 5 minutes for every hour of every day. I am using this code:
```
string cronExpression = "0 7/5 * 1/1 * ? *"; //Every 5 minutes, starting at 7 minutes past the hour
CronExpression cron = new CronExpression(cronExpression);
CronTrigger trigger = TriggerBuilder.Create()
    .WithIdentity("trigger1", "group1")
    .WithSchedule(CronScheduleBuilder.CronSchedule(cron))
    .Build();
```
however Before 00:57, it runs the correct cycle every 5 minutes, but at 00:57, the cycle doesn't run until 01:07 (the correct cycle must be 01:02, which means after 5 minutes from 00:57). ). The problem I noticed is that it repeats at minute 57 of every hour

![Screenshot 2024-01-26 103128](https://github.com/quartznet/quartznet/assets/118602474/38b12756-5471-4e1d-b63e-3000aec96168)

Please help me handle this problem. Thanks!

**Comment by jafin** [comment]
I don't believe you can do that with cron, its calculation boundary in the hour (Start at 7min every 5, repeat everyhour)

Perhaps you can achieve this with a simple schedule, like:

```c#
ITrigger trigger = TriggerBuilder.Create()
    .WithIdentity("trigger3", "group1")
    .StartAt(myTimeToStartFiring) //  set time to 7 minutes past whatever hour.
    .WithSimpleSchedule(x => x
        .WithIntervalInMinutes(5)
        .RepeatForever() 
    .ForJob(myJob) // identify job with handle to its JobDetail itself                   
    .Build();
```
